src/SourceBuild/tarball/content/eng/pipelines/vmr-ci.yml

```diff
@@ -0,0 +1,51 @@
+# This is the main build definition (PR+CI) for dotnet/dotnet
```

Comment:
Would it make sense to name this simply ci.yml?  In this context the vmr prefix doesn't seem necessary.  `ci.yml` is a fairly common naming convention used in the repos.